src/components/cylc/tree/GScanTreeItem.vue

```diff
@@ -45,6 +45,12 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
               style="overflow-wrap: anywhere;"
             >
               {{ node.id }}
+              <template v-if="node.type === 'workflow' && node.node.status !== WorkflowState.STOPPED.name">
+                <br/>
+                <span class="text-grey-lighten-1">
+                  Last update: {{ useTimeAgo(node.node.lastUpdated * 1e3, { showSecond: true, updateInterval: 5e3 }) }}
```

Comment:
Is using exponential notation in this context a standard JS script thing? Or are you playing code-golf? ;p

Reply:
Perfectly standard: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#numeric_literals

Easier to eyeball than counting zeros IMO